queries/glimmer_typescript/injections.scm

```diff
@@ -0,0 +1,22 @@
+; inherits: typescript
+
+; extends
```

Comment:
No.
```suggestion
```

And avoid unnecessary (especially whitespace)  comments.

Reply:
Removed, thanks!

Reply:
`extends` is very much a thing; it's just not appropriate here.